target/linux/ramips/mt7621/base-files/etc/board.d/01_leds

```diff
@@ -78,6 +78,10 @@ netgear,r6800)
 	ucidef_set_led_netdev "lan3" "LAN3" "white:lan3" "lan3"
 	ucidef_set_led_netdev "lan4" "LAN4" "white:lan4" "lan4"
 	;;
+raisecom,msg1500-x.00)
+	ucidef_set_led_netdev "2.4g" "2.4G" "blue:2.4g" "wlan0"
+	ucidef_set_led_netdev "5g" "5G" "blue:5g" "wlan1"
```

Comment:
Please use DT triggers instead.

Reply:
OK. And I add "link" to keep the same logic with the vendor firmware.